tests/ui/specs/ArcSlider/ArcSlider-specs.js

```diff
@@ -0,0 +1,81 @@
+const Page = require('./ArcSliderPage');
+
+describe('ArcSlider', function () {
+
+	beforeEach(function () {
+		Page.open();
+	});
+
+	describe('default', function () {
+		const arcSlider = Page.components.arcSliderDefault;
+
+		it('should change the position of the slider knob on each different click position', function () {
+			const originalCx = parseInt(arcSlider.circle.getCSSProperty('cx').value);
+			const originalCy = parseInt(arcSlider.circle.getCSSProperty('cy').value);
```

Comment:
How about making a function `get knobPosition` that gets `arcSlider.circle.getCssProperty('cx').value` and `arcSlider.circle.getCSSProperty('cy').value` in `ArcSliderInterface`?

Reply:
good idea. I added it